### ISSUE TITLE
Upload Component Status Accessibility Issues

### ISSUE THREAD

**Issue by BrianMitchL** [telerik/kendo-react#1302]
## I'm submitting a...

* Bug report

## Current behavior
The KendoReact Upload component has significant accessibility issues when it comes to screen reader announcements of the start and completion of file transfers.

## Expected behavior
The minimum that must be done is to indicate that the file upload is under way, completed, and its status.

Two WCAG 2.1 Level A success criteria that are not currently met:

1. SC1.1.1 Non-Text Content (Level A)
  - There is no text alternative for the success/failure icon next to "Done"
2. SC4.1.3 Status Updates (Level A)
  - Uploading, Done, and status (success / failure) are not announced to indicate transfer status to screen reader users

## Minimal reproduction of the problem with instructions
Default demo: https://www.telerik.com/kendo-react-ui/components/upload/

## What is the motivation or use case for changing the behavior?
Screen reader users are missing key information with the usability and ability to upload files with the Uploader component.

## Environment

Package versions:
The latest version used on the documentation site.

Browser:
- Chrome (desktop) version 101

System:
- Platform:  Windows

## Suggested Solution

A few small changes to the implementation of the "Uploading… / Done" indicator will provide a minimally adequate solution to these issues.

![image002](https://user-images.githubusercontent.com/3683168/168614189-29437140-2f7b-485d-8652-1246e7af9610.png)

1. Add `aria-live="polite"` to the `<div>` that wraps the "Update/Done" text (and icon)
2. Add off-screen content to the icon so success/failure is also announced with "Done"

Sample code (implementing suggested change 1)

```html
<div class="k-upload-status k-upload-status-total" aria-live="polite">
  <span class="k-i-exception k-icon"></span>
  Done
</div>
```

A quick test was done using the standard [Upload demo](https://www.telerik.com/kendo-react-ui/components/upload/) adding the ARIA live attribute (using Chrome with NVDA) showed it announced "Uploading..." and "Done." 

Adding off-screen text inside or with the final status icon (not the animation spinner) should announce with "Done." These should be short and be reasonably clear as to the outcome.

Possible text alternatives:
- "Successful upload," Announcement would be "Successful upload, Done"
- "Upload Failed," Announcement would be "Upload failed, Done" Ideally, if there were multiple failures, a count would be more descriptive.
